### PR TITLE
The Top submissions should not be including the cancelled ones.

### PR DESCRIPTION
**Pull request by muzaffaryousaf**
Cancelled submissions should not be returned in top_submissions method.

https://openedx.atlassian.net/browse/TNL-1455

### CODE REVIEW

**Comment by muzaffaryousaf** [comment]
@symbolist  can you please review this PR.
Thanks.

File: submissions/api.py [hunk 1]
```diff
@@ -443,16 +443,16 @@ def get_top_submissions(course_id, item_id, item_type, number_of_top_scores, use
...
-            ).select_related("submission").order_by("-points_earned")
+                latest__points_earned__gt=0
+            ).select_related('latest', 'latest__submission').order_by("-latest__points_earned")
 
             if read_replica:
                 query = _use_read_replica(query)
-            scores = query[:number_of_top_scores]
+            score_summaries = query[:number_of_top_scores]
         except DatabaseError:
             msg = u"Could not fetch top scores for course {}, item {} of type {}".format(
```
Comment (symbolist) [inline submissions/api.py]: You may want to update that to 'score summaries'.

File: submissions/tests/test_api.py [hunk 2]
```diff
@@ -283,13 +283,18 @@ def test_get_scores(self):
...
 
-        student_1 = api.create_submission(student_item, "Hello World")
-        student_2 = api.create_submission(student_item, "Hello World")
-        student_3 = api.create_submission(student_item, "Hello World")
+        student_item_3 = copy.deepcopy(STUDENT_ITEM)
+        student_item_3['student_id'] = 'Li'
+
+        student_1 = api.create_submission(student_item_1, "Hello World")
+        student_2 = api.create_submission(student_item_2, "Hello World")
+        student_3 = api.create_submission(student_item_3, "Hello World")
```
Comment (symbolist) [inline submissions/tests/test_api.py]: Can you also add multiple submissions for one student and ensure that only the latest one shows?

**Comment by symbolist** [comment]
Otherwise :+1:

**Comment by muzaffaryousaf** [comment]
@stephensanchez  can you please review this PR. 
Thanks.

**Comment by stephensanchez** [comment]
Please update the docstring on the API function, otherwise this looks good! :+1:

**Pull request merged**